dev-tools/docker/docker_entrypoint.sh

```diff
@@ -23,7 +23,7 @@ cd "$MY_DIR/../.."
 # Note: no need to clean due to the .dockerignore file
 
 # Build the code
-make -j`grep -c '^processor.*[1-6]$' /proc/cpuinfo` # TODO remove the processor hack
+make -j`grep -c '^processor' /proc/cpuinfo` # TODO remove the processor hack
```

Comment:
Is the comment for removing the processor hack obsolete? Or that's different?

Reply:
snap!